lib/CleantalkSP/SpbctWP/AdminBannersModule/AdminBanners/AdminBannerAbstract.php

```diff
@@ -11,11 +13,37 @@ public function getCurrentScreenId()
         return get_current_screen()->id;
     }
 
-    abstract public function show();
+    public function show()
```

Comment:
It's not obvious what this method do. Maybe the correct name should be "showIfNeeded" ? Please, rename or refactor.

Reply:
No, It's fine now